modules/central-state/README.md

```diff
@@ -0,0 +1,27 @@
+# @phenyl/central-state
+
+
+```js
+import { MongoClient } from 'mongodb'
```

Comment:
This code seems mongo-db example. We can't use central-state itself without db client.
Empty example would be better, I think.